gruntfile.js

```diff
@@ -63,15 +164,89 @@ module.exports = function (grunt) {
                 }
 
                 // If the tests have their own tsconfig, add that as a new target
-                // if (grunt.file.exists(modulePath + '/test/tsconfig.json')) {
-                //     buildCmds.ts[key + '-tests'] = {
-                //         'tsconfig': modulePath + "/test/tsconfig.json",
-                //     };
-                // } else if (grunt.file.exists(modulePath + '/tests/tsconfig.json')) {
-                //     buildCmds.ts[key + '-tests'] = {
-                //         'tsconfig': modulePath + "/tests/tsconfig.json",
-                //     };
-                // }
+                var addQunit = false;
+                var testRoot = "" 
+                if (modules[key].testHttp !== false) {
+                    testRoot = "http://localhost:9001/" 
+                }
+
+                var testUrl = testRoot + modulePath + "/test/UnitTests.html";
+                if (grunt.file.exists(modulePath + '/test/tsconfig.json')) {
```

Comment:
Refactoring the tests grunt definition (already done this for internal variant).